test/integration/servicebus/queue.test.js

```diff
@@ -18,9 +18,10 @@ describe('ServiceBus', function() {
       test.client = new AMQPClient(Policy.ServiceBusQueue);
     });
 
-    afterEach(function() {
+    afterEach(function(done) {
       return test.client.disconnect().then(function() {
         test.client = undefined;
+        done();
```

Comment:
If you return the promise, the promise is guaranteed to be resolved as part of the mocha step. See "Working with Promises" over here: https://mochajs.org/